_api-reference/index-apis/flush.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: default
+title: Flush
+parent: Index APIs
+nav_order: 36
+---
+
+# Flush
+
+**Introduced 1.0**
+{: .label .label-purple }
+
+The Flush API manually flushes in-memory data from an index onto disk. This ensures that any memory from the index stored in the buffer persists on disk, as opposed to writing all buffer data to the disk, which can affect the performance of your cluster.
```

Comment:
```suggestion
The Flush API takes all in-memory operations and stores them to segments on disk. Operations flushed to an index segment are no longer needed for transactions logs to recover in the event of a cluster restart. 

OpenSearch automatically performs flushes in the background based on certain conditions such as the size of the transaction log, controlled by the `index.translog.flush_threshold_size` setting. However, performing a manual flush operation can improve performance.
```

@sohami and @mgodwan: Thoughts?

Reply:
1. `are no longer needed **for** transactions logs` to `are no longer needed **in** transactions logs` ?
2. `However, performing a manual flush operation can improve performance` I don't think this is true in all the cases, so we should avoid generalizing it.

Reply:
@sohami: Why would someone need to manually flush their segments?

Reply:
We can leave it out if it makes things more confusing, that being said, I do think the flush mechanism would benefit from the additional context you're providing.